### ISSUE TITLE
Time series split during inner cross validation

### ISSUE THREAD

**Issue by VGODIE** [awslabs/autogluon#1004]
I've searched the documentation and failed to find a possibility to use option to make inner validation not randomly based on some 'time' parameter - in my case data is time dependent and i want to assure that during validation step only data 'from past' is used as train set, and 'from future' as validation set. 

Is there any option to enable it ?

**Comment by Innixma** [comment]
We are currently working on this functionality. The PR of the WIP time-series support is available here: #684 

You can refer to the existing issue tracking this feature request here: #617 

At present, it's recommended to not use bagging / stacking for time series problems in AutoGluon and instead use a train/val split and pass both into fit as `train_data`, `tuning_data`. Once #684 is merged, the plan is to investigate adding the inner cross-val time-series support for time-series ensembling.

I'll close this as duplicate for now.

**Comment by jeremyhermann** [comment]
I see that #684 was merged and that you now support time series forecasting. Is there now a way to do temporal splits (eg something like sklearn.model_selection.TimeSeriesSplit) with the TabularPredictor?
